lib/cooked_post_processor.rb

```diff
@@ -51,14 +51,16 @@ def grant_badges
   end
 
   def keep_reverse_index_up_to_date
-    upload_ids = Set.new
+    upload_ids = Array.new
 
     @doc.css("a/@href", "img/@src").each do |media|
       if upload = Upload.get_from_url(media.value)
         upload_ids << upload.id
       end
     end
 
+    upload_ids.concat(oneboxed_image_uploads.pluck(:id))
```

Comment:
We were using a set previously so I think you'll need to make sure this is unique. I believe you can do something like 

`upload_upds |= oneboxed_image_uploads.pluck(:id)`